parsers/xml2ram.go

```diff
@@ -28,14 +29,19 @@ func ParseAlphabet(folderPath string) ([]dictionary.Entry, error) {
 			}
 			mut.Lock()
 			dico = append(dico, e...)
-			fmt.Printf("entries in %s: %d\n", string(r), len(e))
 			mut.Unlock()
 		}(r)
 	}
 	wg.Wait()
 	if wgErr != nil {
 		return nil, wgErr
 	}
+
+	// save it, for future reference
+	err := saveGob(dico)
+	if err != nil {
+		return nil, fmt.Errorf("unable to save gob: %w", err)
```

Comment:
I think that an error here should be returned but not block the rest of the process.